client/terraform_cli_test.go

```diff
@@ -148,8 +148,42 @@ func TestTerraformCLIInit(t *testing.T) {
 			}
 
 			assert.NoError(t, err)
+			m.AssertExpectations(t)
 		})
 	}
+
+	t.Run("workspace artifact with empty state", func(t *testing.T) {
+		// Edge case to handle https://github.com/hashicorp/terraform/issues/21393
+		initErr := errors.New(`Initializing the backend...
```

Comment:
Does tfexec/terraform fall back into an interactive mode here? That seems.. strange.

Reply:
Yeah it's a Terraform bug where it errors with an interactive message but TF is ran in automation with tfexec which disables input.

Reply:
That being said, i'll include the GH issue where the logic is above in addition to this test case to keep context collocated